source/features/remove-label-with-a-single-click.tsx

```diff
@@ -0,0 +1,84 @@
+import './remove-label-with-a-single-click.css';
+import React from 'dom-chef';
+import XIcon from 'octicon/x.svg';
+import select from 'select-dom';
+import oneTime from 'onetime';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import onReplacedElement from '../helpers/on-replaced-element';
+import {getRepoURL, getConversationNumber} from '../github-helpers';
+
+const canEditLabels = oneTime((): boolean => select.exists('.sidebar-labels .octicon-gear'));
+
+function updateSidebar() {
+	select('#partial-discussion-sidebar')!.dispatchEvent(new CustomEvent('socket:message', {
+		bubbles: false,
+		cancelable: false,
+		detail: {
+			name: '',
+			data: {},
+			cached: false
+		}
+	}));
+}
+
+async function removeButtonClickHandler(event: delegate.Event<MouseEvent, HTMLSpanElement>): Promise<void> {
+	event.preventDefault();
+
+	const removeButton = event.delegateTarget;
```

Comment:
Is it a `closeButton` or `removeButton`? Let's be consistent everywhere in this file